plugin/conflict_marker.vim

```diff
@@ -120,7 +120,7 @@ endfunction
 
 augroup ConflictMarkerDetect
     autocmd!
-    autocmd BufReadPost,FileChangedShellPost,ShellFilterPost,StdinReadPost,BufEnter,FocusGained,ColorScheme * if conflict_marker#detect#markers()
+    autocmd BufReadPost,FileChangedShellPost,ShellFilterPost,StdinReadPost * if conflict_marker#detect#markers()
```

Comment:
This will break highlighting when refreshing colorscheme (see #11)

Reply:
As pointed out in the commit message, using `:hi-default` fixes that issue.

Reply:
Oh, I see. I didn't think that this change is linked to the above change.